selection_sort/Python/selection_sort.py

```diff
@@ -0,0 +1,17 @@
+def selection_sort(arr):
+    for i in range(len(arr)):
+        min_i = min(arr[i:]) 
+        min_index = arr[i:].index(min_i) 
+        arr[i + min_index] = arr[i] 
+        arr[i] = min_i
+    sorted_arr = arr
+    return sorted_arr
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpxipfo4mc/selection_sort/Python/selection_sort.py
+++ b/tmp/tmpxipfo4mc/selection_sort/Python/selection_sort.py
@@ -7,6 +7,7 @@
     sorted_arr = arr
     return sorted_arr
 
+
 def main():
     arr = [2, 3, 0, 4]
     sorted_arr = selection_sort(arr)
```